flux-dsl/src/main/java/com/influxdb/query/dsl/Flux.java

```diff
@@ -1373,6 +1416,29 @@ public final RangeFlux range(@Nonnull final Long start, @Nonnull final Long stop
         return new RangeFlux(this).withStart(start, unit).withStop(stop, unit);
     }
 
+    /**
+     * Filters the results by time boundaries.
+     *
+     * @param start Specifies the oldest time (Unix timestamp in seconds) to be included in the results
+     * @param stop  Specifies the exclusive newest time (Unix timestamp in seconds) to be included in the results
+     * @return {@link RangeFlux}
+     */
+    @Nonnull
+    public final RangeFlux range(final long start, final long stop) {
+        return new RangeFlux(this).withStart(start).withStop(stop);
+    }
+
+    /**
+     * Filters the results by time boundaries.
+     *
+     * @param start Specifies the oldest time (Unix timestamp in seconds) to be included in the results
+     * @return {@link RangeFlux}
+     */
+    @Nonnull
+    public final RangeFlux range(final long start) {
```

Comment:
Please use `Long` object:

```suggestion
    public final RangeFlux range(final Long start) {
```